README.md

```diff
@@ -51,74 +51,13 @@ task-mappings create EVERGREEN_PROJECT_NAME
 ```
 Currently, it can only analyze public git repos. Private repo support is coming in a future version.
 
-Its options are described below.
```

Comment:
Removed this options documentation because having it here is duplicative (it is provided by running `task-mappings create --help`) and easily falls out of date.